.env

```diff
@@ -1,6 +1,22 @@
 REACT_APP_TIMEOUT=1000
```

Comment:
This files needs to be deleted. or shouldn't be pushed.

Reply:
if you have any conf updates. Please update the .env.example file

Reply:
if you have any conf updates. Please update the .env.example file

Reply:
removed